### PR TITLE
ignore signatures that don't do anything...

### PR DESCRIPTION
**Pull request by maxtaco**
but no need to crash key reconstruction

### CODE REVIEW

**Comment by maxtaco** [comment]
@jacobhaven  can you take a look?

**Comment by maxtaco** [comment]
or @oconnor663 ...

File: openpgp/keys.go [hunk 1]
```diff
@@ -441,7 +441,17 @@ EachPacket:
...
+				//
+				// See https://github.com/keybase/client/issues/2666
+				//
+				// There might have been a user attribute picture before this signature,
+				// in which case this is still a valid PGP key. In the future we might
+				// not ignore user attributes (like picture). But either way, it doesn't
+				// make sense to bail out here. Keep looking for other valid signatures.
+				//
+				// Used to be:
+				//    return nil, errors.StructuralError("signature packet found before user id packet")
```
Comment (oconnor663) [inline openpgp/keys.go:454]: Is there any way to log the fact that we're dropping this? Silent drops have been confusing for us before.

Comment (maxtaco) [inline openpgp/keys.go:454]: We'd have to add more fields like `BadSubkeys` into the `Entity` class. Doable...

**Comment by oconnor663** [comment]
:ship: assuming adding logging above is more trouble than it's worth.

**Pull request merged**